app/models/miq_server.rb

```diff
@@ -152,6 +152,7 @@ def start
     #############################################################
     # Start all the configured workers
     #############################################################
+    clean_heartbeat_files
```

Comment:
Thanks @NickLaMuro I though about that but didn't think it would hurt to just do it blindly since it's only done once at server startup.